cases.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
Did you intend to commit this file?

Reply:
I've fixed it. It was part of my explorations as to how better automate the removal of `cases'`.